liquidity/liquidity.go

```diff
@@ -152,6 +165,14 @@ type Parameters struct {
 	// dispatched swaps in our current budget, inclusive.
 	AutoFeeStartDate time.Time
 
+	// AutoOut is a boolean that indicates whether the liquidity manager
+	// should dispatch loop out swaps automatically.
+	AutoOut bool
+
+	// MaxAutoInFlight is the maximum number of in-flight automatically
+	// dispatched swaps we allow.
+	MaxAutoInFlight int
```

Comment:
Unused in this commit. I also thought you would never do more than one, because of uncertainty about the final fee?